### DISCUSSION TITLE
Getting Started: Modeling knickpoint Migration from an Increase in Stream Power

### DISCUSSION THREAD

**Discussion by camersiegNCSU** [csdms/project#9] [Q&A]
Hello,
I am very new to using land lab and modeling in general so any advice or help would be greatly appreciated. I am looking to model a past landscape's response to an increase in stream power, starting at approximately 1 Ma, which would stem from the lowest point in the drainage network. This would be done in order to test if a knickpoint I see in the present-day drainage network was created from that increase in stream power. How would I get started? Any help would be greatly appreciated!

**Comment by gregtucker** [comment]
Hi! Here are some suggestions.

First, it's a good idea to be familiar with the theory that you hope to use. Whipple (2004) Bedrock Rivers and the Geomorphology of Active Orogens (if I'm remembering the title right) is a good place to start. There have been lots of papers since then on erosion laws for long-term fluvial erosion; some authors to look for, for example, include Turowski, Lague, Kirby, among many others... these are just a few. Once you get into the literature, you'll start to see the thread of papers on erosion laws for rivers.

On landscape evolution modeling generally, I'm obviously biased but I like Tucker & Hancock (2010) in ESPL, which cites several other useful review papers.

On numerical methods in geoscience, Kump and Slingerland's book Mathematical Modeling of Earth's Dynamical Systems is a gem. Jon Pelletier also has a great book on numerical modeling with applications to geomorphology.

Then to learn Landlab, I recommend starting with the introductory tutorials. Then move on to the "teaching notebooks" that address landscape evolution.

Hopefully others will weigh in too but that's at least a starting point.
